bundles/framework/divmanazer/component/VisualizationForm.js

```diff
@@ -129,6 +129,97 @@ Oskari.clazz.define(
             return values;
         },
 
+        /**
+         * @public @method getOskariStyle
+         * Returns Oskari JSON style
+         *
+         * @return {JSON}
+         */
+        getOskariStyle: function () {
+            const values = this.getValues();
+            var oskariStyle = {
+                featureStyle: {
+                    fill: {
+                        color: values.dot.color,
+                        area: {
+                            color: values.area.fillColor,
+                            pattern: values.area.fillStyle
+                        }
+                    },
+                    stroke: {
+                        color: values.line.color,
+                        width: values.line.width,
+                        lineDash: values.line.style,
+                        lineCap: values.line.cap,
+                        lineJoin: values.line.corner,
+                        area: {
+                            color: values.area.lineColor,
+                            width: values.area.lineWidth,
+                            lineDash: values.area.lineStyle,
+                            lineJoin: values.area.lineCorner
+                        }
+                    },
+                    text: {
+                        labelText: values.dot.message
+                    },
+                    image: {
+                        shape: values.dot.shape,
+                        size: values.dot.size
+                    }
+                }
+            };
+            return JSON.stringify(oskariStyle);
```

Comment:
This could return the style object instead of a string.

Reply:
And maybe just return the featurestyle part. Other than the added parts like lineDash etc is this something that can be passed to mapmodule.getStyle()?

Reply:
It seems that lineDash is expected to be an array in mapmodule.getStyle(), but since it is a string in visualization form it cannot be used unless the cases would be transformed somehow. Dot shape and area fill pattern are also problematic.

I was also thinking that maybe getOskariStyle() should return an object with oskari style JSONs separately for dot, line and area styles instead of trying to pack everything together?